src/Discovery/aggMDSUtils.jsx

```diff
@@ -114,4 +114,15 @@ const loadStudiesFromAggMDS = async () => {
   return studies;
 };
 
-export default loadStudiesFromAggMDS;
+const loadAuthMappingsFromWTS = async () => {
+  const res = await fetch(`${hostname}wts/aggregate/authz`);
```

Comment:
Do we even need `hostname` here? Could we do `/wts/...`? Not sure what the end URL looks like, but I'm assuming it's referring to the portal's own host, right?

Reply:
usually we set a new variable for a path in `localconfs` and import into each individual components, rather than directly importing `hostname` everywhere